metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -511,3 +501,81 @@ def _convert_value(param):
         "(run-id *{run_id}*).".format(name=obj.workflow_name, run_id=run_id),
         bold=True,
     )
+
+
+@argo_workflows.command(help="Delete the Argo workflow")
+@click.option(
+    "--authorize",
+    default=None,
+    type=str,
+    help="Authorize the deletion with a production token",
+)
+@click.pass_obj
+def delete(obj, authorize=None):
+    validate_token(obj.workflow_name, obj.token_prefix, obj, authorize, "delete")
+    obj.echo("Deleting workflow *{name}*...".format(name=obj.workflow_name))
+    schedule_deleted = ArgoWorkflows.remove_schedule(obj.workflow_name)
+    if schedule_deleted:
```

Comment:
Should we add a short comment explaining why it's OK to just swallow schedule deletion failure?  I am guessing for these reasons:
* We want to proceed to template deletion, even if `CronWorkflow` already got deleted some other way (or by someone else).
* Users might run this twice (after seeing message that it might take a while, etc.)
* Even if `CronWorkflow` deletion fails and the object sticks around, it is not a big deal, as long as WorkflowTemplate gets deleted below, anyway.

In both cases we don't want to worry the end user.  I used up some brain juice to consider it... can spell it out for future developers!